fetch.js

```diff
@@ -292,6 +292,10 @@
       throw new RangeError('Invalid status code')
     }
 
+    var link = document.createElement('a')
```

Comment:
This is failing inside service worker because `document` is unavailable. Any suggestions? https://travis-ci.org/github/fetch/jobs/84728696